src/System.Collections/src/System/Collections/Generic/LinkedList.cs

```diff
@@ -188,7 +188,7 @@ public void CopyTo(T[] array, int index)
 
             if (index < 0 || index > array.Length)
```

Comment:
IMO it does detract from the readability but not insofar that it becomes confusing.